docs/api-docs/payments/payments-api-overview.md

```diff
@@ -591,6 +591,7 @@ Orders can be created using the [Server to Server API Endpoints](https://develop
 | `30104` | The payment was declined. Please contact card issuer for more information. |N/A |N/A|
 | `30105` | The payment was declined due to duplicate payment being submitted. |N/A |N/A |
 | `30106` | The payment was declined due to insufficient funds. |N/A |N/A|
+| `30107` | The payment was declined due to stored instrument no longer being valid. |PayPal Buyer Revoked Pre-Approved Payment Authorization|N/A|
```

Comment:
How about following for the possible causes?
`Shopper revoked payment authorization associated to the stored PayPal account.`

Reply:
Sure, I've updated the copy.